brave/brave-ui
# Adds type attribute to grantClaim (ads)

Fixes: https://github.com/brave/brave-ui/issues/341

## Changes
Adds a `type` and optional `amount` attribute (For using the BAT amount as part of the message for ads) to the `grantClaim` component. Dual grant claim notifications are added in to the settings concept page.

## Test plan
1. Use Now Link to navigate to both the Desktop and Mobile settings concept pages
2. Ensure both the UGP grant and Ads grant notifications show and can be interacted with
3. Navigate to the grantClaim individual component feature.
4. Toggle between UGP/Ads type to ensure both are reflected.

##### Link / storybook path to visual changes
https://brave-ui-102ehcyjd.now.sh
- 

Desktop Preview:
<img width="575" alt="screen shot 2019-01-08 at 3 16 19 am" src="https://user-images.githubusercontent.com/8732757/50824473-ecc1fd80-12f3-11e9-957d-ed8831caa098.png">

Mobile Preview:
<img width="423" alt="screen shot 2019-01-08 at 3 17 59 am" src="https://user-images.githubusercontent.com/8732757/50824522-0cf1bc80-12f4-11e9-96b9-d25b1428177d.png">




## Integration
  
     - When uplift-approved, merge to brave-core-0.VV.x feature branch
     - Create additional brave-core PRs for each feature branch to update commit
